RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+This release fixes an issue where a GET request is processed despite it being disallowed.
```

Comment:
Let's be more specific here
```suggestion
This release changes the default behavior on queries via GET requests to a 405 error. To enable queries via get, please use `self.allow_queries_via_get`.
```

Reply:
Hey @erikwrede I appreciate your reviewing this. I agree the message can be improved, but let's be clear about the change. Queries via GET are still allowed by default just like before - I did not touch that, though I believe the default should be more conservative. I changed the behavior when it's turned off: 

- before this change you get a GraphQL query not found error;
- after this change you get a HTTP method not allowed error.

Reply:
@alexei so probably we can go with an explanation like that, saying that before, even though queries via GET are allowed by default, when disallowing them you would only get a GraphQL query not found error, but now you should get a 405 error instead.